src/main/java/com/tstepnik/planner/domain/user/userregistrationdto/RegisterUserMapper.java

```diff
@@ -0,0 +1,18 @@
+package com.tstepnik.planner.domain.user.userregistrationdto;
+
+import com.tstepnik.planner.domain.user.User;
+import org.mapstruct.Mapper;
+
+
+import java.util.List;
+
+@Mapper(componentModel = "spring")
+public interface RegisterUserMapper {
+
+    RegisterUserDto toDto(User user);
```

Comment:
I don't think we'll ever need changing user back to this dto. He should only be used for registration